templates/contrib/jdbc/README.md

```diff
@@ -34,14 +34,14 @@ The drivers images are based on the supported configurations for Red Hat Process
 |-------------|-------------------------|---------------------------------|
 | IBM DB2     | 11.1                    | jcc-11.1.4.4.jar                |
 | MariaDB     | 10.2                    | mariadb-java-client-2.3.0.jar   |
-| MS SQL      | 2014,2016               | mssql-jdbc-7.0.0.jre8.jar       |
+| MS SQL      | 2014,2016               | mssql-jdbc-7.2.2.jre8.jar       |
```

Comment:
mssql-jdbc-7.2.2.jre8.jar should be mssql-jdbc-7.2.2.jre11.jar to match what is in prepare.sh

Reply:
Oops, yes, missed jre11, thanks.